app/config/message-queue.js

```diff
@@ -49,4 +51,4 @@ if (error) {
   throw new Error(`The message queue config is invalid. ${error.message}`)
 }
 
-module.exports = value
+export default value
```

Comment:
Might be worth throwing this one out to the wider team @aaroncarroll82 @bendef-24 & @omega1119  but if we are setting out best practices is it worth us switching to named exports. They keep the name of the exported thing explicit throughout the app rather than allowing the importer to choose a name, which may be inconsistent.

With default exports `import value from './config'` & `import config from `./config` are the same thing with different names. A named export would force `import { value } from `./config`.

Reply:
I also prefer named exports rather than default exports - happy to make this change assuming nobody else minds?